starter-repos/app-template/config/envs/dev/kustomization.yaml

```diff
@@ -12,9 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Customize the namespace to be the namespace name used for dev environment.
-# The namespace shouldn't be changed once it is customized.
-namespace: dev
+# ACTION REQUIRED: Replace the $USER with the namespace created in step #9.
+# https://github.com/GoogleCloudPlatform/solutions-modern-cicd-anthos/blob/master/docs/index.md#testing-the-change-locally-with-skaffold
+namespace: $USER
```

Comment:
Actually we don't need to specify the namespace for dev? In the user guide, the user runs:
```
kubectl create namespace $USER
kubens $USER
```

Which sets the namespace to $USER, and it will be used when they run `skaffold dev`. Would you verify if that works? If so, we don't need to specify namespace here anymore.

Reply:
I tried, but it looks like it will use the default namespace "redis" instead of the ones applied by kubens